Simulator/augments.py

```diff
@@ -0,0 +1,52 @@
+from enum import Enum, auto
```

Comment:
You should be working in the Set12Simulator folder instead of the Simulator folder. I pushed a new branch yesterday that included that new simulator folder.